systemtest/src/main/java/io/strimzi/systemtest/Constants.java

```diff
@@ -410,7 +410,7 @@ public interface Constants {
 
     // main namespace for Cluster Operator deployment
     String CO_NAMESPACE = "co-namespace";
-    String TEST_SUITE_NAMESPACE = "test-suite-namespace";
+    String TEST_SUITE_NAMESPACE = Environment.isNamespaceRbacScope() ? CO_NAMESPACE : "test-suite-namespace";
```

Comment:
I don't think this is a proper place for it. This line will change Constants file into a file where something is evaluated. Wouldn't be better to move the decision to Abstract or to Env?

Reply:
AbstractsST class is not feasible because we use such value outside of tests so I can move it to the Environment class if that's okay with you?